src/Controls/tests/DeviceTests/Elements/Window/WindowTests.Windows.cs

```diff
@@ -171,5 +218,19 @@ public async Task MinimizeAndThenMaximizingWorks()
 				Assert.Equal(2, deactivated);
 			}
 		}
+
+		Task PerformClick(IButton button)
```

Comment:
This should probably be in the shared assert extensions for reuse in other tests. Android has this feature as well, so we can do better test share.

Reply:
Is not necessary to create the test. We can do the same without the Button.